### PR TITLE
build: use Bazel 1.1.0

### PR DESCRIPTION
**Pull request by lizan**
Signed-off-by: Lizan Zhou <lizan@tetrate.io>

### CODE REVIEW

File: bazel/foreign_cc/BUILD [hunk 1]
```diff
@@ -11,7 +11,10 @@ configure_make(
         "--enable-shared=no",
         "--enable-frame-pointers",
         "--disable-libunwind",
-    ],
+    ] + select({
```
Comment (lizan) [inline bazel/foreign_cc/BUILD]: @keith I believe this is workaround for https://github.com/bazelbuild/bazel/pull/9154

Comment (keith) [reply bazel/foreign_cc/BUILD]: We should update rules_foriegn_cc instead where this is fixed

Comment (keith) [reply bazel/foreign_cc/BUILD]: We should fix it there too then 😞

File: bazel/io_opentracing_cpp.patch [hunk 2]
```diff
@@ -1,16 +1,36 @@
+diff --git a/mocktracer/BUILD b/mocktracer/BUILD
```
Comment (htuch) [inline bazel/io_opentracing_cpp.patch]: Can we get this fixed upstream? CC @g-easy

Comment (g-easy) [reply bazel/io_opentracing_cpp.patch]: CC @rnburn for OpenTracing ;)

Comment (lizan) [reply bazel/io_opentracing_cpp.patch]: https://github.com/opentracing/opentracing-cpp/pull/120

**Comment by lizan** [comment]
@htuch let's not block on the upstream patches?

**Comment by htuch** [comment]
@lizan I'm inclined as a general principle to make a best effort to merge patches upstream first. Not a huge fan of building up patches. But, if you think this is time sensitive, or is blocking something else, I'm cool with making that async.

**Comment by lizan** [comment]
@htuch agreed on upstream first, but I'd also like to get bazel up-to-date before stable release.

**Pull request merged**